### ISSUE TITLE
Ensures every ARIA input field has an accessible name.

### ISSUE THREAD

**Issue by Ram-G** [Azure/LogicAppsUX#4455]
### Describe the issue 

1. Open designer to an action that uses the HTML editor. 
2. Observe "editor-input" field does not have an accessible name. 

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

![image](<https://github.com/Azure/LogicAppsUX/assets/10837129/9f1e4943-dad1-4dad-9cf8-a3c780fa6a89>)

### Browser 

Edge

### Additional context 

ADO link: [https://msazure.visualstudio.com/One/\_workitems/edit/27260245](<https://msazure.visualstudio.com/One/_workitems/edit/27260245>)

**Comment by Eric-B-Wu** [comment]
Fixed in https://github.com/Azure/LogicAppsUX/pull/4810
